paraclear/README.md

```diff
@@ -0,0 +1,73 @@
+# Paraclear Account Value Checker
+
+This repository contains a Python script designed to interact with a Starkware-based smart contract to retrieve account value information, including perpetual asset balances. Specifically, it retrieves margin health checks data for an account on Paradex
+
+## Features
+
+- Retrieves account value
+- Checks account health (Initial Margin and Maintenance Margin)
+- Lists perpetual balances for available markets
+
+## Prerequisites
+
+- **Python 3.7+**
+- **pip**: (Python package manager)
+- **Starknet.py**: The script uses the `starknet_py` library to interact with the StarkNet blockchain.
+- **Requests**: To fetch markets data from an API.
+- **Rich**: For fancy terminal printing.
+
+## Setup
+
+1. Clone this repository:
+   ```
+   git clone <repository-url>
+   cd <repository-directory>
+   ```
+
+2. Install the required dependencies:
+   ```
+   pip install -r requirements.txt
+   ```
+
+3. Set up the following environment variables:
+   - `BASE_API_URL`: Base URL for the API
+   - `FULL_NODE_URL`: URL of the StarkNet full node
+   - `PARACLEAR_ADDRESS`: Address of the Paraclear contract
+   - `PRDX_API_URL`: URL for the PRDX API
+   - `L2_ADDRESS`: The StarkNet L2 address you want to check
```

Comment:
We need to not say starknet here. This would only work for Paradex addresses so we should changes that.